### PR TITLE
Do better at separating ourselves from The Carpentries

### LINKED ISSUE #34: Clarify Footer
* ***About HPC Carpentry***: rephrase to make it clear we operate *in the spirit of The Carpentries*, but not as an official program. (This is excerpted from the main About Us page, I think.)
* ***Links***: 
  - [ ] Create a landing page for *our* Code of Conduct, with instructions to contact a Steering Committee member to report a breach. Then link out to The Carpentries' policy.
  - [ ] Create a landing page for *our* Handbook, or drop it.
  - [ ] Create a landing page for *our* Privacy Policy, with instructions to contact a Steering Committee member to report a breach. Then link out to The Carpentries' policy.
  - [x] Drop Annual Reports.
  - [ ] Create a new section for Carpentries Links to redirect to our sister programs.

### PR DESCRIPTION
**Pull request by ocaisa**
Fixes #34

### CODE REVIEW

**Comment by ocaisa** [comment]
@tkphd Ok, now I get why some pages use html and some markdown. Basically if you want fancy layouts you need to use html (like when injecting tweets like in the last commit)

File: pages/carpentries.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: page-fullwidth
+title: "The Carpentries"
+permalink: /carpentries/
+---
+
+HPC Carpentry aspires to become an official Carpentries program...but we are not there yet! Meanwhile, we link
```
Comment (tkphd) [inline pages/carpentries.md]: Please limit lines to 99 characters or fewer.

File: pages/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page-fullwidth
+title: "HPC Carpentry Code of Conduct"
+permalink: /code-of-conduct/
+---
+
+We ask that people adhere to the guidelines of
```
Comment (tkphd) [inline pages/code-of-conduct.md]: Let's provide an overview of our Code. Suggestion follows.

---

We are dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity.  In order to foster a positive and professional learning environment we encourage the following practices:

* Use welcoming and inclusive language.
* Be respectful of different viewpoints and experiences.
* Gracefully accept constructive criticism.
* Focus on what is best for the community.
* Show courtesy and respect towards other community members.

Our community has chosen to adopt the official [Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html). If you have questions about the policy, concerns over observed behaviours, or need to report a violation of the Code, we ask that you reach out to any member of the [HPC Carpentry Steering Committee](/contact#steering). Your communications will be held in confidence. We pledge to acknowledge your message within 24 hours, and to take action within 30 calendar days.

File: pages/privacy.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+layout: page-fullwidth
+title: "HPC Carpentry Privacy Policy"
+permalink: /privacy/
+---
+
```
Comment (tkphd) [inline pages/privacy.md:6]: This needs work. The privacy policy relates to information we collect and store, which we have full control over. Currently, we do not collect any sensitive personally identifiable information; we ask for names of instructors & helpers, but no email or social media handles, so those can be considered ambiguous. Let's spell out this stance, and for now, *not* adhere to The Carpentries' policy.

File: _data/network.yml [hunk 4]
```diff
@@ -2,29 +2,14 @@
...
-  title: "Our Reports"
-
-- name: "Software Carpentry website"
-  url: "https://software-carpentry.org/"
-  title: "Software Carpentry"
-
-- name: "Data Carpentry website"
-  url: "http://www.datacarpentry.org/"
-  title: "Data Carpentry"
+- name: "The Carpentries"
```
Comment (tkphd) [inline _data/network.yml]: Having a link called "The Carpentries" on our front page can be misinterpreted as membership. Perhaps "Resources" or "Related Organizations" would be better?

**Review by tkphd — CHANGES_REQUESTED** [review]
Looks good overall. Some suggestions for content modification.

File: pages/privacy.md (hunk 3 above)
Comment (ocaisa) [reply pages/privacy.md:6]: Ok, I made a stab at it.

**Comment by ocaisa** [comment]
@tkphd This should be good to go now, I did my best with the 99 char limit but URLs make that impossible sometimes.

**Comment by ocaisa** [comment]
I've also emptied the dummy form responses, so they will disappear when this gets merged (we need to encourage people to register their workshops!).

File: pages/carpentries.md (hunk 1 above)
Comment (ocaisa) [reply pages/carpentries.md]: Did my best!

File: pages/code-of-conduct.md (hunk 2 above)
Comment (ocaisa) [reply pages/code-of-conduct.md]: Thanks

File: _data/network.yml (hunk 4 above)
Comment (ocaisa) [reply _data/network.yml]: "Other resources" now

File: pages/contact.md [hunk 5]
```diff
@@ -3,22 +3,27 @@ layout: page-fullwidth
...
-<head>
+<div class="row">
+<div class="medium-8 columns">
+<h2 id="steering">
 Steering Committee
-</head>
+</h2>
 <p>
-HPC Carpentry is lead by a steering committee. The members help to foster the HPC Carpentry organization and its lessons, guide curriculum development following The Carpentries Handbook, and participate in discussions with The Carpentries in a representative capacity to help us navigate the pathway to induction as an official Lesson Program.
+HPC Carpentry is lead by a steering committee. The members help to foster the HPC
```
Comment (tkphd) [inline pages/contact.md]: lead -> led

File: pages/privacy.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+Currently, we do _require_ any sensitive personally identifiable information: providing
```
Comment (tkphd) [inline pages/privacy.md]: "we do require" -> "we do not require"

**Review by tkphd — CHANGES_REQUESTED** [review]
Awesome! A couple of picayune changes, but this is good to go once the critical "not" is inserted.

**Review by tkphd — APPROVED** [review]

**Comment by tkphd** [comment]
Nice work, @ocaisa.

**Pull request merged**